LeezeIt/LeezeIt/TextField.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  TextField.swift
+//  LeezeIt
+//
+//  Created by Riffat Khan on 9/6/22.
+//
+
+import Foundation
+import UIKit
+
+// adding max lenght attribute on interface builder
+private var __maxLengths = [UITextField: Int]()
+
+extension UITextField {
+    @IBInspectable var maxLength: Int {
+        get {
+            guard let l = __maxLengths[self] else {
```

Comment:
your story says "Restrict the user to enter more than 6 letters in password." change the logic of maxLength

Reply:
Resolved